docs/zh_CN/large-bucket/README.md

```diff
@@ -37,11 +37,11 @@ minio server http://host{1...4}/export{1...16}
 ```
 
 ### 3. 验证设置是否成功
-要验证是否部署成功，你可以通过浏览器或者[`mc`](https://docs.minio.io/docs/minio-client-quickstart-guide)来访问刚刚部署的Minio服务。你应该可以看到上传的文件在所有Minio节点上都可以访问。
+要验证是否部署成功，你可以通过浏览器或者[`mc`](https://docs.minio.io/cn/minio-client-quickstart-guide)来访问刚刚部署的Minio服务。你应该可以看到上传的文件在所有Minio节点上都可以访问。
 
 ## 了解更多
-- [mc快速入门](https://docs.minio.io/docs/minio-client-quickstart-guide)
-- [使用 aws-cli](https://docs.minio.io/docs/aws-cli-with-minio)
-- [使用 s3cmd](https://docs.minio.io/docs/s3cmd-with-minio)
-- [使用 minio-go SDK](https://docs.minio.io/docs/golang-client-quickstart-guide)
-- [Minio文档](https://docs.minio.io)
\ No newline at end of file
+- [mc快速入门](https://docs.minio.io/cn/minio-client-quickstart-guide)
+- [使用 aws-cli](https://docs.minio.io/cn/aws-cli-with-minio)
+- [使用 s3cmd](https://docs.minio.io/cn/s3cmd-with-minio)
+- [使用 minio-go SDK](https://docs.minio.io/cn/golang-client-quickstart-guide)
+- [Minio文档](https://docs.minio.io)
```

Comment:
Should https://docs.minio.io be changed to https://docs.minio.io/cn/ ?

Reply:
Yes,You are right about it.

Reply:
ping @jingsam  can the links be updated.